hack/build/common.sh

```diff
@@ -15,7 +15,14 @@
 CDI_DIR="$(readlink -f $(dirname $0)/../../)"
 BIN_DIR=${CDI_DIR}/bin
 OUT_DIR=${CDI_DIR}/_out
+TESTS_OUT_DIR=$OUT_DIR/tests
 CMD_OUT_DIR=${OUT_DIR}/cmd
 BUILD_DIR=${CDI_DIR}/hack/build
 MANIFEST_TEMPLATE_DIR=${CDI_DIR}/manifests/templates
 MANIFEST_GENERATED_DIR=${CDI_DIR}/manifests/generated
+
+function build_func_tests() {
+    mkdir -p ${TESTS_OUT_DIR}/
+    ginkgo build ${CDI_DIR}/tests
```

Comment:
Is there a benefit in compiling the test binary manually before execing it?  This limits tests to only being run in linux environments for one.

Why not instead execute `go test ./tests/... -args -ginkgi.v`?

Reply:
Because we are not executing the tests at this point, in hack/build/functests.sh is where we are running the actual tests. At that point we are passing in a bunch of parameters like the path to kubectl and kubeconf.

Reply:
Right.  What I mean is - why not run the above command in hack/build/functest.sh rather than compiling separately first?  Is there anything gained by compiling them beforehand?

Reply:
That is the way kubevirt was doing it and I copied it (its a theme :)). But there is no reason. so I can change it.